package-examples/helloworld-kustomize/Kptfile

```diff
@@ -0,0 +1,9 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: helloworld-kustomize
```

Comment:
Not clear to me, why are we have `kustomize` in the name here ?

Reply:
I am keeping the name more or less the same as it was.  Happy to entertain a different name.  I think there is a question of what this should be named in general, but right now I am focused on just making sure that the old samples can use the new format.

Reply:
Ack.